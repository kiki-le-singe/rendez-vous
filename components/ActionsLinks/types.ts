export type ActionsLinksProps = React.PropsWithChildren<{}>;
