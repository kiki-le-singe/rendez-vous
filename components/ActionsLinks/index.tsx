import { StyleSheet, View } from "react-native";

import { ActionsLinksProps } from "./types";

export default function ActionsLinks({ children }: ActionsLinksProps) {
  return <View style={styles.actionsLink}>{children}</View>;
}

const styles = StyleSheet.create({
  actionsLink: {
    flexDirection: "row",
    gap: 18,
  },
});
