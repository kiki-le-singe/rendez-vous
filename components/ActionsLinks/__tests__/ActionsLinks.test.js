import * as React from "react";
import renderer from "react-test-renderer";
import { View } from "react-native";

import ActionsLinks from "..";

it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <ActionsLinks>
        <View />
      </ActionsLinks>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
