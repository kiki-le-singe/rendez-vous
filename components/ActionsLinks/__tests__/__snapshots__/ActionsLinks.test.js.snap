// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "flexDirection": "row",
        "gap": 18,
      },
      {},
    ]
  }
>
  <View />
</View>
`;
